frontend/src/components/question-creation/question-elements/TextElement.tsx

```diff
@@ -33,6 +33,7 @@ const TextElement = ({ id, data }: TextElementProps): React.ReactElement => {
 
   return (
     <Textarea
+      sx={{ fontSize: "18px", fontWeight: "400", lineHeight: "23px" }}
```

Comment:
couldn't get `textStyle` to work here... so manually added typography 😟 let me know if you have thoughts on how to use `textStyle` to work instead!

Reply:
hmm maybe you could look into defining variants in `themes/components/textArea.ts`?